.trivyignore

```diff
@@ -1 +1,17 @@
-
+CVE-2022-24775
```

Comment:
I think these CVEs can be removed :)

Reply:
i tried to remove them but the CI jobs are failing - https://github.com/canonical/wordpress-k8s-operator/actions/runs/12081279416/job/33690383343